h/static/scripts/account/auth-controller.coffee

```diff
@@ -18,7 +18,12 @@ class AuthController
       $scope.form?.$setPristine()
 
     failure = (form, response) ->
-      {errors, reason} = response.data
+      try
+        {errors, reason} = response.data
+      catch
+        form.responseErrorMessage = "Oops, something went wrong on the server.
```

Comment:
I think you want to say `reason =` because `formRespond` sets `responseErrorMessage` along with marking the form as invalid.